include/pyre3.14t/lock.h

```
@@ -0,0 +1,59 @@
+typedef struct PyMutex {
+    uint8_t _bits;  /* (private) */
+} PyMutex;
+
+PyAPI_FUNC(void) PyMutex_Lock(PyMutex *m);
+PyAPI_FUNC(void) PyMutex_Unlock(PyMutex *m);
+PyAPI_FUNC(int) PyMutex_IsLocked(PyMutex *m);
+
+/* Takes the mutex, parking the calling thread until it is free.  A parked
+   thread gives the GIL up, so whoever holds the mutex can run. */
+static inline void
+_PyMutex_Lock(PyMutex *m)
```

Clang (14.0.6)

[warning] declaration uses identifier '_PyMutex_Lock', which is a reserved identifier; cannot be fixed because 'PyMutex_Lock' would conflict with a macro definition

(bugprone-reserved-identifier)